### PR TITLE
NEPT-1129: Implements the checkboxes component.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

**Comment by Fefaine** [comment]
:+1:

File: templates/checkboxes/checkboxes.component.inc [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    $variables['small_description'] = array(
+      '#markup' => t('List of options for @title', array('@title' => $title)),
+      '#theme_wrappers' => array('html_tag'),
+      '#tag' => 'p',
+      '#attributes' => $descr_attributes_array,
+    );
+  }
+
+  if (!empty($element['#description'])) {
+    $variables['description'] = $element['#description'];
```
Comment (voidtek) [inline templates/checkboxes/checkboxes.component.inc]: what is "$element['#description']" ? 
This variable is not used in the tpl.

**Comment by drupol** [comment]
Just rebased the PR to have #8 goodies.

**Comment by imanoleguskiza** [comment]
👍

**Pull request merged**